deep_quoridor/src/renderers/training_status.py

```diff
@@ -0,0 +1,37 @@
+from agents.core import AbstractTrainableAgent
+from renderers import Renderer
+
+
+class TrainingStatusRenderer(Renderer):
+    def __init__(self, update_every: int, total_episodes: int, agents: list[AbstractTrainableAgent]):
+        self.agents = agents
+        self.update_every = update_every
+        self.total_episodes = total_episodes
+        self.episode_count = 0
+
+    def start_game(self, game, agent1, agent2):
+        self.step = 0
+        self.player1 = agent1.name()
+        self.player2 = agent2.name()
+
+    def end_game(self, game, result):
+        if self.episode_count % self.update_every == 0:
+            for agent in self.agents:
+                agent_name = agent.name()
+                avg_reward, avg_loss = agent.compute_loss_and_reward(self.update_every)
```

Comment:
I moved this class and the only change was moving this function to AbstractTrainableAgent so it can be called from other parts